specification/immunisation-fhir-api.yaml

```diff
@@ -514,11 +514,11 @@ components:
       name: -diseaseType
       description: |
         The vaccine preventable disease the dose is being administered against.
-        Expressed as `<value>` where`<value>` must be a valid Snomed Code.
+        Expressed as `<value>` where`<value>` must be a valid Disease Type (COVID19, FLU, HPV, MMR).
```

Comment:
?Should be vaccine type, not disease type

Reply:
as discussed, this is because the query parameter is diseaseType